src/components/common/Onboarding/Wizard/WizardSidebarItem.tsx

```diff
@@ -41,12 +45,22 @@ const WizardSidebarItem = ({
         <div
           className={clsx('h-2.5 w-2.5 rounded-full', {
             'bg-base-white':
-              (currentStep >= stepId && hasSubItems) ||
-              (!hasSubItems && currentStep === stepId),
-            'border border-base-white': currentStep < stepId,
+              ((currentStep >= stepId && hasSubItems) ||
+                (!hasSubItems && currentStep === stepId)) &&
```

Comment:
Maybe we could make a `const isCompletedStep` or something

Reply:
All right will do :)